### ISSUE TITLE
html-to-react crashes on invalid style tag

### ISSUE THREAD

**Issue by nickpalmer** [aknuds1/html-to-react#52]
```
function createStyleJsonFromString(styleString) {
  styleString = styleString || '';
  var styles = styleString.split(/;(?!base64)/);
  var singleStyle, key, value, jsonStyles = {};
  for (var i = 0; i < styles.length; ++i) {
    singleStyle = styles[i].split(':');
    if (singleStyle.length > 2) {
      singleStyle[1] = singleStyle.slice(1).join(':');
    }
    
    key = camelize(singleStyle[0]);
    value = singleStyle[1];
    //
    // If key or value are undefined this blows up
    //
    if (key.length > 0 && value.length > 0) {
      jsonStyles[key] = value;
    }
  }
  return jsonStyles;
}
```

**Comment by nickpalmer** [comment]
Would love to get this merged and up on npm ASAP. :) Thanks for a great project.

**Comment by aknuds1** [comment]
Merged your PR and published version 1.3.1, thanks! Next time, please put "Fixes #<issue number>" in your PR's description as it will close the corresponding issue automatically.
